docs/testing.md

```diff
@@ -43,7 +49,51 @@ Or if you want to run bats from a different repository or branch, edit `centos7-
         bats_forklift_repo: https://github.com/<YOUR_NAME>/forklift.git
         bats_forklift_version: your-branch
 
-## Pipeline Testing
+### To run Bats on an existing VM for development
+
+1. Clone forklift onto your VM (it's recommended to clone into a directory named other than `forklift` so you don't get mixed up):
+```
+git clone git@github.com:theforeman/forklift.git forklift-devel
+```
+2. Clone the Bats repo:
+```
+git clone https://github.com/sstephenson/bats.git
+```
+3. Install Bats on your VM in the place where `katello-bats` expects it:
+```
+cd bats
+sudo ./install.sh /opt/bats
+```
+4. Make sure `bats` is in your `PATH`:
+```
+export PATH="/opt/bats/bin:$PATH"
+```
+5. Make sure the `bats` command works:
+```
+bats
+Bats 0.4.0
+Usage: bats [-c] [-p | -t] <test> [<test> ...]
+```
+6. Instead of using forklift's `bats/install.sh` to copy our Bats tests, create symlinks instead:
```

Comment:
`install.sh` is another thing that should die. It's unused in our pipelines anyway.